docs/contributing/code/patches.rst

```diff
@@ -131,10 +131,9 @@ Work on your Patch
 Work on the code as much as you want and commit as much as you want; but keep
 in mind the following:
 
-* Practice :doc:`BDD <bdd>`, which is the development methodology we use at Sylius;
+* Practice :doc:`BDD </bdd/index>`, which is the development methodology we use at Sylius;
```

Comment:
```suggestion
* Practice :doc:`BDD </bdd/index>`, which is the development methodology we use in Sylius;
```
it both `at` and `in` but here we're talking about the framework, not the company 😂